newtab.html

```diff
@@ -6,17 +6,26 @@
     <link rel="stylesheet" href="/aos-master/dist/aos.css" />
     <link rel="stylesheet" type="text/css" href="style.css">
     <script type="text/javascript" src="jquery-3.3.1.min.js"></script>
-    <script type="text/javascript" src="scripts.js"></script>
     <script src="/aos-master/dist/aos.js"></script>
+    <script type="text/javascript" src="scripts.js"></script>
   </head>
   <body>
-    <div class = "fluid-container">
-        <div class = "main"></div>
-        <div class= "quote" data-aos="fade-left"><p>"Travel changes you. As you move through this life and this world you change things slightly, you leave marks behind, however small. And in return, life and travel leaves marks on you. Most of the time, those marks on your body or on your heart are beautiful. Often though, they hurt"</p></div>
-        <div id="results" data-aos = "slide-up"></div>
-        <div id="ytplayer"><iframe id="ytplayer" type="text/html" width="100%" height="390"
-    src="https://www.youtube.com/embed/M7lc1UVf-VE?autoplay=1&origin=https://example.com"
-    frameborder="0"/></div>
+    <div class="fluid-container">
+      <div class="main"></div>
+      <div class="quote" data-aos="fade-left">
+        <p>"Travel changes you. As you move through this life and this world you change things slightly, you leave marks behind, however small. And in return, life and travel leaves marks on you. Most of the time, those marks on your body or on your heart are beautiful. Often though, they hurt"</p>
+      </div>
+      <div id="results" data-aos="slide-up"></div>
+      <div id="ytplayer">
+        <iframe
```

Comment:
HTML supports multi-line and looks much cleaner if you have a lot of attributes on an element.